### PR TITLE
docs: changelog 4.24.8

### PR DESCRIPTION
**Pull request by MadCcc**
- 🌐 Add missing translation for `pt_BR`. [#40019](https://github.com/ant-design/ant-design/pull/40019) [@rafaelncarvalho](https://github.com/rafaelncarvalho)
- TypeScript
  - 🤖 Fix `CompoundedComponent` type error from external module. [#39058](https://github.com/ant-design/ant-design/pull/39058) [@wangcch](https://github.com/wangcch)

---

- 🌐 补全 `pt_BR` 国际化。[#40019](https://github.com/ant-design/ant-design/pull/40019) [@rafaelncarvalho](https://github.com/rafaelncarvalho)
- TypeScript
  - 🤖 修复部分组件 `CompoundedComponent` 类型报错。[#39058](https://github.com/ant-design/ant-design/pull/39058) [@wangcch](https://github.com/wangcch)

[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -149,7 +149,7 @@
     "rc-steps": "~5.0.0-alpha.2",
     "rc-switch": "~3.2.0",
     "rc-table": "~7.26.0",
-    "rc-tabs": "~12.5.0",
+    "rc-tabs": "~12.5.6",
```
Comment (afc163) [inline package.json:152]: 这个 changelog 也可以写一下。

**Comment by MadCcc** [comment]
@SocketSecurity ignore puppeteer@19.6.3

**Comment by yoyo837** [comment]
#40626

File: package.json (hunk 1 above)
Comment (afc163) [reply package.json:152]: https://github.com/ant-design/ant-design/blob/master/package.json

5.x 也升级一下。

https://github.com/ant-design/ant-design/issues/39999#issuecomment-1425077956

**Review by li-jia-nan — APPROVED** [review]

**Pull request merged**